src/renderer/hooks/useKeystoreClientStates.ts

```diff
@@ -13,7 +13,7 @@ import { liveData } from '../helpers/rx/liveData'
 
 export type KeystoreClientStates = RD.RemoteData<Error, boolean>
 
-export const useKeystoreClientStates = (): { clientStates: KeystoreClientStates } => {
+export const useKeystoreClientStates = (): { clientStates: RD.RemoteData<Error, boolean> } => {
```

Comment:
Is this type changed by accident maybe?

Reply:
Good catch! I can't remember the reason for that change - I will revert it.